src/main/app/package-lock.json

```diff
@@ -1,8 +1,14465 @@
 {
```

Comment:
Heb je een clean install gedaan van het project? Misschien is dat nog nodig vanwege de angular upgrade

Reply:
Had voorheen nog geen clean install gedaan.
Inmiddels wel.